### PR TITLE
Copy getJtreg to shared location

### PR DESCRIPTION
**Pull request by annaibm**
-Moved the getJtreg download logic to TKG/scripts/getDependencies.xml

resolves : https://github.com/adoptium/aqa-tests/issues/4848

### CODE REVIEW

**Comment by annaibm** [comment]
Grinder outputs:
openjdk: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36142/
Crypto test: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36150/
ssl-test:https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36151

File: functional/security/Crypto/build.xml [hunk 1]
```diff
@@ -51,81 +51,14 @@
...
-			</else>
-		</if>
-	</target>
+
+	<!-- Call getJtregVersion, getJtreg target from getDependencies.xml -->
+    <target name="invokeGetJtreg" depends="getJtregVersion,getJtreg">
+    </target>
 	
-	<target name="dist" depends="getDependentLibs,getJtreg,getCryptoTest" description="generate the distribution">
+	<target name="dist" depends="invokeGetJtreg,getCryptoTest" description="generate the distribution">
```
Comment (llxia) [inline functional/security/Crypto/build.xml:61]: Just call `getJtreg` instead and remove line 57 - 59. In `getJtreg` (TKG repo), it depends on `getJtregVersion`.

**Review by llxia — CHANGES_REQUESTED** [review]

**Comment by annaibm** [comment]
Grinder outputs:
openjdk: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36330/
Crypto test: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36331/
ssl-test:https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36332/

File: openjdk/openjdk.mk [hunk 2]
```diff
@@ -178,6 +178,7 @@ ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
 	TEST_VARIATION_JIT_PREVIEW:=-XX:-JITServerTechPreviewMessage
 	TEST_VARIATION_JIT_AGGRESIVE:=-Xjit:enableAggressiveLiveness
 	TIMEOUT_HANDLER:=-timeoutHandler:jtreg.openj9.CoreDumpTimeoutHandler -timeoutHandlerDir:$(Q)$(LIB_DIR)$(D)openj9jtregtimeouthandler.jar$(Q)
+	EXTRA_OPTIONS := -Xverbosegclog $(EXTRA_OPTIONS)
```
Comment (llxia) [inline openjdk/openjdk.mk:181]: Please remove unrelated change.

**Comment by llxia** [comment]
Please also run openjdk test with hotspot.

**Comment by annaibm** [comment]
Grinder Tests:
openjdk with hotspot: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36373/

**Comment by llxia** [comment]
> openjdk: https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/36330/

Console output:

```
00:01:47.001  getJtregVersion:
00:01:47.001  
00:01:47.001  getDependentLibs:
00:01:47.001       [exec] --------------------------------------------
00:01:47.001       [exec] path is set to /home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/lib
00:01:47.001       [exec] task is set to default
00:01:47.001       [exec] dependencyList is set to jtreg_7_3_1_1,tohandler_simple
00:01:47.001       [exec] --------------------------------------------
00:01:47.001       [exec] Starting download third party dependent jars
00:01:47.001       [exec] --------------------------------------------
00:01:47.001       [exec] downloading dependent third party jars to /home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/lib
00:01:47.001       [exec] /home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/lib/openj9jtregtimeouthandler.jar exists with correct hash, not downloading
00:01:47.001       [exec] /home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/lib/jtreg_7_3_1_1.tar.gz exists with correct hash, not downloading
00:01:47.001       [exec] downloaded dependent third party jars successfully
00:01:47.001  
00:01:47.001  getJtreg:
00:01:47.001      [mkdir] Created dir: /home/jenkins/workspace/Grinder/jvmtest/openjdk
00:01:47.001       [copy] Copying 1 file to /home/jenkins/workspace/Grinder/aqa-tests/openjdk
00:01:47.449  
00:01:47.449  openjdk-jdk.check:
```

**Review by llxia — APPROVED** [review]
LGTM

**Comment by llxia** [comment]
This PR has to be merged together with https://github.com/adoptium/TKG/pull/473

**Review by smlambert — APPROVED** [review]
TKG PR was merged earlier today, bringing this one in as they are related.

**Pull request merged**